### PR TITLE
Styled select

### PR DESCRIPTION
**Pull request by danielle-dsouza**
Added styled-select component and changed theme colours

### CODE REVIEW

File: frontend/src/components/atoms/Select.tsx [hunk 1]
```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import styled from "styled-components";
+
+export interface SelectProps {
```
Comment (xsharonhe) [inline frontend/src/components/atoms/Select.tsx:4]: find base interface: div -> interface IProjectsProps extends React.HTMLAttributes<HTMLDivElement> {};

File: frontend/src/components/atoms/Select.tsx [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+export const Select: React.FC<SelectProps> = ({
+    options,
+    children,
+    ...props
+}): React.ReactElement => (
+    <Wrapper {...props}>
+        <StyledSelect>
+            <SOption value='' hidden>{children}</SOption>
+            {options.map((option, index) => (
```
Comment (xsharonhe) [inline frontend/src/components/atoms/Select.tsx:16]: remove index it is not needed lol my bad

**Review by xsharonhe — COMMENTED** [review]
Small changes

File: frontend/src/components/atoms/Select.tsx [hunk 3]
```diff
@@ -1,23 +1,53 @@
...
+    options: string[];
+}
+
+export const Select: React.FC<SelectProps> = ({
+  hidden,
+  options,
+  children,
+  ...props
+}): React.ReactElement => (
+  <Wrapper>
```
Comment (danielle-dsouza) [inline frontend/src/components/atoms/Select.tsx:15]: <Wrapper {...props> results in a no overload call error

File: frontend/src/components/atoms/Select.tsx [hunk 4]
```diff
@@ -1,23 +1,53 @@
+import React from "react";
 import styled from "styled-components";
 
-export const Select = styled.select`
-    ${({ theme }) => `
+export interface SelectProps extends React.HTMLAttributes<HTMLSelectElement> {
+    hidden?: boolean;
```
Comment (danielle-dsouza) [inline frontend/src/components/atoms/Select.tsx:5]: Hidden is fixed on chrome

**Pull request merged**